lms/templates/vert_module.html

```diff
@@ -1,10 +1,31 @@
 <%page expression_filter="h"/>
-<%! from openedx.core.djangolib.markup import HTML %>
+
+<%namespace name='static' file='/static_content.html'/>
+
+<%!
+from django.utils.translation import gettext as _
+
+from openedx.core.djangolib.markup import HTML
+%>
 
 %if unit_title and show_title:
   <h2 class="hd hd-2 unit-title">${unit_title}</h2>
 % endif
 
+% if graded:
```

Comment:
We might want to add this even if the content isn't graded. As long as the section has a due date, it'll prevent submission, even if it's just a practice problem.

Reply:
@Ahernkb What do you think? The original ticket said graded, but I'm happy to update.

Reply:
Thanks, Dillon. I agree with Cale here - if it has a due date and missing it will disable the submit button, it's helpful to display the due date.